lib/destination/utils/load.go

```diff
@@ -55,9 +55,9 @@ func DataWarehouse(cfg config.Config, store *db.Store) destination.DataWarehouse
 		}
 		return s
 	case constants.BigQuery:
-		return bigquery.LoadBigQuery(cfg, store)
```

Comment:
Can we explicitly pass in `nil` for  `store` on line 52 where we do `snowflake.LoadSnowflake(cfg, store)`?